### ISSUE TITLE
Fluid typography does not scale up to 200% - Fails 1.4.4 Resize text Level AA 

### ISSUE THREAD

**Issue by LukaszJaro** [WordPress/gutenberg#43488]
### Description

This was first brought up in the comments in this article https://make.wordpress.org/themes/2022/08/15/testing-and-feedback-for-the-fluid-typography-feature/

I can confirm, I tested this with FSE, latest gutenberg version. I added the new fluid keys, while they work great for responsiveness the text actually **shrinks** the more you zoom in on chrome. When not using clamp this does not happen and text grows up to 200%.

https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=144#resize-text

Unfortunately there is no zoom api that can be implemented to switch off fluid typography and automated accessibility scanning tools won't pick up on this either.

There is a github issue related to this https://github.com/w3c/csswg-drafts/issues/6869

### Step-by-step reproduction instructions

1. Add new fluid keys in theme,json
2. Create new post
3. Add regular h2 with no clamp
4. Add h2 with clamp and fluid size
5. Publish results and view on front-end
6. Zoom in using chrome and watch or measure text(100% - 500%). Also observe differences between clamp and non clamp h2

### Screenshots, screen recording, code snippet

Chrome 500% zoom.
h2 no clamp set as 2rem. 
h2 clamp max set as 2rem and min as 1.5rem.

![image](https://user-images.githubusercontent.com/10998875/185956433-70ee4527-9771-4db7-a456-cfcb0d0e7891.png)

### Environment info

- Custom theme, Gutenberg 13.9.0, WordPress 6.0.1
- Chrome, windows 10 

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes
